File: app/src/app/services/api/openapi/models/kontakt-data.ts
/* tslint:disable */
/* eslint-disable */
import { Kontakt } from './kontakt';
import { OsebaData } from './oseba-data';
import { SporociloData } from './sporocilo-data';
export interface KontaktData {
  kontakt?: Kontakt;
  oseba_refs?: Array<OsebaData>;
  sporocilo_posiljatelj_refs?: Array<SporociloData>;
  sporocilo_prejemnik_refs?: Array<SporociloData>;
}

File: api/core/src/main/kotlin/core/data/KontaktData.kt
package core.data

import core.domain.Kontakt
import kotlinx.serialization.Serializable

@Serializable
data class KontaktData(
    val kontakt: Kontakt,
    val oseba_refs: List<OsebaData> = listOf(),
    val sporocilo_prejemnik_refs: List<SporociloData> = listOf(),
    val sporocilo_posiljatelj_refs: List<SporociloData> = listOf()
)
